app/services/user-language.js

```diff
@@ -0,0 +1,67 @@
+import Ember from 'ember';
+import C from 'ui/utils/constants';
+import { ajaxPromise } from 'ember-api-store/utils/ajax-promise';
+
+export default Ember.Service.extend({
+  prefs            : Ember.inject.service(),
+  session          : Ember.inject.service(),
+  intl             : Ember.inject.service(),
+  translationsList : Ember.computed.alias('app.translations'),
+  loadedLocales    : null,
+
+  bootstrap: function() {
+    this.set('loadedLocales', []);
+  }.on('init'),
+
+  initLanguage() {
+    const upl           = this.getLanguage(); // get language from user prefs
+    const uplLocal      = this.get('session').get(C.PREFS.LANGUAGE); // get local language
+    let defaultLanguage = C.LANGUAGE.DEFAULT;
+
+    if (upl) { // if user prefs
+      if (upl !== defaultLanguage) {
```

Comment:
Remove the `if`s?  If it is the same setting it is harmless (like the `// no user pref language` block)